README.md

```diff
@@ -1,2 +1,33 @@
 # jpo-asn-pojos
+
 Generated POJOs from ASN SAE and IEEE standards
+
+These are Java codecs for XER and JER ASN.1 encodings.
+
+Folders:
+
+* [jpo-asn-runtime](jpo-asn-runtime/README.md) - Runtime library.
+* [jpo-asn-j2735-2024](jpo-asn-j2735-2024/README.md) - POJOs for the J2735/2024 specification.
+* [jpo-asn-test-generator](jpo-asn-test-generator/README.md) - A Java command-line utility for
+  generating test XER and JER messages.
+* [generate-test-data](generate-test-data/README.md) - Various utilities for generating test data
+  and examples of serialized
+  messages.
+
+Prerequisites:
+
+* JDK 21
+
+To build using Gradle:
+
+```bash
+cd jpo-asn-j2735-2024
+./gradlew clean build
```

Comment:
question: I assume this will be added in a subsequent PR, but there isn't a mvnw file under the j2735 directory and running `mvn clean package` there fails

Reply:
Maven works differently from gradle.  There is a composite pom at the root level for Maven, but not for gradle.

The instructions say to run "./gradlew clean build" from the jpo-asn-j2735-2024 directory, which succeeds for me (even with no classes in that project).

They also say to run "./mvnw clean package" not from the subdirectory, but from the root directory, which also succeeds for me.

To build only one project using the mvnw file you can do something this from the root:

./mvnw -pl jpo-asn-runtime -am clean package